tests/RobotFramework/tests/config_management/child_conf_mgmt_plugin.robot

```diff
@@ -228,6 +224,8 @@ Custom Setup
     Execute Command    find . -type f -name "c8y-configuration-plugin_*.deb" -exec dpkg -i {} \\;
 
     ${parent_ip}=    Get IP Address
+    Execute Command    sudo tedge config set http.bind.address ${parent_ip}
+    Restart Service    tedge-agent
```

Comment:
@reubenmiller I'm not quite sure I understand how this test ever worked. It's clearly expecting the child device to connect to the file transfer service on the main device, but the bind address was seemingly never being set?

Reply:
Wow, very confusing...but I think I understand it now...These tests were only working because the test cases are executed sequentially, and the first test case `Prerequisite Parent` does a `tedge connect/disconnect` on the parent which was previously restarting the tedge-agent.

Reply:
I'll change it to restart the agent where it was previously doing so in that case